$NetBSD: patch-at,v 1.3 2010/03/16 10:59:09 tnn Exp $

Treat DragonFly like FreeBSD.
Avoid silly redefinitions of standard types on NetBSD.

--- mozilla/gfx/qcms/qcmstypes.h.orig	2009-08-24 17:52:41.000000000 +0200
+++ mozilla/gfx/qcms/qcmstypes.h
@@ -10,6 +10,8 @@
 #if defined (__SVR4) && defined (__sun)
 /* int_types.h gets included somehow, so avoid redefining the types differently */
 #include <sys/int_types.h>
+#elif defined(__NetBSD__)
+#include <stdint.h>
 #else
 typedef PRInt8 int8_t;
 typedef PRUint8 uint8_t;
@@ -23,7 +25,7 @@ typedef PRUint64 uint64_t;
 #ifdef __OS2__
 /* OS/2's stdlib typdefs uintptr_t. So we'll just include that so we don't collide */
 #include <stdlib.h>
-#elif defined(__FreeBSD__)
+#elif defined(__FreeBSD__) || defined(__DragonFly__)
 /* FreeBSD typedefs uintptr_t in /usr/include/sys/types.h */
 #else
 typedef PRUptrdiff uintptr_t;
